server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/documents.ts

```diff
@@ -80,7 +87,8 @@ export function create(
             ensureSingleUseToken: true,
             singleUseTokenCache,
         }),
-        throttle(throttler, winston, commonThrottleOptions),
+        throttle(tenantThrottler, winston, commonThrottleOptions),
+        throttle(clusterThrottler, winston, createDocThrottleOptions),
```

Comment:
we need to swap the order here. Cluster throttling check before tenant throttling.

Reply:
Oh, missed it. Thanks. Done. :)